envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/envoy/EnvoyContainer.kt

```diff
@@ -39,7 +39,7 @@ class EnvoyContainer(
         private const val ADMIN_PORT = 10000
 
         private const val MIN_SUPPORTED_ENVOY_VERSION = "v1.22.7"
```

Comment:
is it still true?

Reply:
![image](https://github.com/allegro/envoy-control/assets/138656232/3ce43960-99fc-45e1-a0b8-f96a49a5b620)

Reply:
this version was there since first time introducing it into the repo - I don't know what was the logic behind setting it and I don't see anything in `java-control-plane` about compatibility